src/OrchardCore.Modules/OrchardCore.Queries/Sql/SqlParser.cs

```diff
@@ -785,6 +806,59 @@ private void EvaluateOverClauseOptional(ParseTreeNode overClauseOpt)
             _builder.Append(")");
         }
 
+        private string EvaluateCteStatement(ParseTreeNode cteStatement)
+        {
+            var builder = new StringBuilder();
```

Comment:
Looks like it could use `_builder` instead of creating another one

Reply:
definitely, I've just fixed it